frontend/packages/dev-console/src/components/topology/TopologySideBar.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { ModelessOverlay } from 'patternfly-react';
+import { TopologySideBar as PfTopologySideBar } from '@patternfly/react-topology';
```

Comment:
Console convention is to use all uppercase for acronyms in camel case names

```suggestion
import { TopologySideBar as PFTopologySideBar } from '@patternfly/react-topology';
```